docs/howtos/layers/vectors.md

```diff
@@ -11,30 +11,85 @@ kernelspec:
-programmatically and from the GUI.
+microscopy. You can adjust the color, width, and length of all the vectors from
+the console or from the GUI.
+
+## Using the GUI to manipulate vectors
+
+Before you can use the GUI to manipulate vectors, you must load a vector layer.
+Please refer to [A simple example](#a-simple-example) to add a `vectors` layer
+first, then explore the editing options the GUI provides.
+
+* Opacity - click and hold the circle on the opacity slider bar and adjust it to
+  any value between 0.00 (clear) and 1.00 (completely opaque).
+* Width - adjusting the width makes the vectors appear thicker or thinner. Use
+  the + and - buttons on either side of the width bar to adjust width or click
+  on the number in the middle of the bar and enter a value. The minimum value is
+  0.10.
+* Length - adjusting the length makes the vector longer or shorter. Use the +
+  and - buttons on either side of the length bar to adjust length or click on
+  the number in the middle of the bar and enter a value. The minimum value is
+  0.10.
+* Blending - `blending` has the options of `opaque`, `translucent`,
+  `translucent no depth`, `minimum`, or `additive` in the dropdown. Refer to the
+  [Blending layers](blending-layers) section of _Layers at a glance_ for an
+  explanation of each type of blending.
+* Edge color mode - select `direct`, `cycle`, or `colormap` from the dropdown.
+  * Direct (default mode) allows each vector to be set arbitrarily.
+  * Cycle allows the color to be set via a color cycle over an attribute.
+  * Colormap allows the color to be set via a color map over an attribute.
+* Edge color - click the thumbnail next to `edge color:` to select or create a
+  color from the pallette.
+* Out of slice - if this box is checked, `out of slice` is on or true. If this
+  box is not checked, `out of slice` is off or false. If it is on or true,
+  vectors slightly out of slice are rendered.
+
+### Vector starting position  
+
+The starting position of vectors cannot be edited from the GUI. Nor is it
+possible to draw vectors from the GUI. If you want to draw lines using the GUI
+you should use the `Lines` shape inside a `shapes` layer.
+
+### 3D rendering
+
+All layers can be rendered in both 2D and 3D. One of the viewer buttons at the
+bottom of the left panel can toggle between these 2 modes.
+When in 2D, the button looks like this: ![image: 2D/3D button](../../images/3D-button.png), ready to switch to 3D mode.
+When in 3D, the button looks like this: ![image: 2D/3D button](../../images/2D-button.png), ready to switch to 2D mode.
+
+The number of dimensions sliders will be 2 or 3 less than the total number of
+dimensions of the layer, allowing you to browse volumetric timeseries data and
+other high dimensional data. See [](../../gallery/nD_vectors) to see shapes in
+both 2D and 3D:
+
+![image: nD vectors](../../images/nD_vectors.webm)
 
 ## A simple example
```

Comment:
I think I would prefer this section about the `GUI controls...`, so right after `When to use...`
I think it makes the GUI controls more clear when you can see the example in the video.?